app/features/IdentitySlice.ts

```diff
@@ -58,6 +61,15 @@ export async function loadIdentities(dispatch: Dispatch) {
     dispatch(updateIdentities(identities));
 }
 
+export async function removeIdentityAndInitialAccount(
+    dispatch: Dispatch,
+    identityId: number
+) {
+    await removeIdentityAndInitialAccountInDatabase(identityId);
+    await loadAccounts(dispatch);
+    return loadIdentities(dispatch);
```

Comment:
This in the nitpicky area, but as the 2 are decoupled, you could do:

```suggestion
    return Promise.all([loadAccounts(dispatch), loadIdentities(dispatch)]);
```